styles.css

```diff
@@ -584,6 +589,45 @@ nav li a img {
   color: white;
 }
 
+.mobile-menu {
+  min-height: 718px;
+  width: 0;
+  display: none;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  background: #0e23dda9;
+  backdrop-filter: blur(5px);
+  transition: 0.6s;
+}
```

Comment:
- [ ] Kindly for a perfect responsive design, It would be better to set the width of your ```mobile-menu```  to ```100vw``` to ensure that the mobile menu takes up the full width of the viewport on all the screens.  😎😎
  
 **Currently, it doesn't take full width on screens larger than 375px👇** 
![image](https://user-images.githubusercontent.com/102757126/217044620-8187b22d-a429-425e-aeab-94de4956025f.png)

you may follow this [link](https://www.educba.com/css-viewport/#:~:text=Definition%20of%20CSS%20Viewport,size%20as%20the%20original%20Webpage.) for more info 😂

Reply:
Width set to 100 vw  as suggested by the code reviewer